### PR TITLE
Updates heading text to Sentence case.

### PR DESCRIPTION
**Pull request by StevenDufresne**
See #306.

This PR lowercases the view titles and heading to sentence case to match wp.org.

### CODE REVIEW

File: settings/src/components/screen-navigation.js [hunk 1]
```diff
@@ -32,14 +32,7 @@ const ScreenNavigation = ( { screen, children, title = '', canNavigate = true }
...
 
-			<h3>
-				{ title.length
-					? title
-					: screen
-							.replace( '-', ' ' )
-							.replace( 'totp', 'Two-Factor Authentication' )
-							.replace( 'webauthn', 'Two-Factor Security Key' ) }
-			</h3>
+			<h3>{ title }</h3>
```
Comment (adamwoodnz) [inline settings/src/components/screen-navigation.js:35]: ❤️

**Comment by adamwoodnz** [comment]
I'm still seeing uppercase due to the [CSS](https://github.com/WordPress/wporg-two-factor/blob/trunk/settings/src/components/screen-navigation.scss#L24)

**Comment by adamwoodnz** [comment]
Also how does one enter the onboarding flow? I've tried `/profile/security`, but `is_onboarding_page()` starts off true then becomes false, so I end up in the normal screens.

**Comment by StevenDufresne** [comment]
> I'm still seeing uppercase due to the [CSS](https://github.com/WordPress/wporg-two-factor/blob/trunk/settings/src/components/screen-navigation.scss#L24)

A good catch, I missed committing that.

**Comment by StevenDufresne** [comment]
> Also how does one enter the onboarding flow? I've tried /profile/security, but is_onboarding_page() starts off true then becomes false, so I end up in the normal screens.

You're probably getting stuck because you have 2fa configured: https://github.com/WordPress/wporg-two-factor/blob/9cac7f8f9cd98a382be303beef544c3bcc159632/settings/settings.php#L233

**Review by adamwoodnz — APPROVED** [review]
Conflicts, otherwise LGTM

**Pull request merged**